google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -294,17 +309,73 @@ void ListAppProfiles(google::cloud::bigtable::InstanceAdmin instance_admin,
 }
 //! [list app profiles]
 
+//! [delete app profile]
+void DeleteAppProfile(google::cloud::bigtable::InstanceAdmin instance_admin,
+                      int argc, char* argv[]) {
+  if (argc < 3) {
+    throw Usage{
+        "delete-app-profile: <project-id> <instance-id> <profile-id>"
+        " [ignore-warnings (default: true)]"};
+  }
+  google::cloud::bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
+  google::cloud::bigtable::AppProfileId profile_id(ConsumeArg(argc, argv));
+  bool ignore_warnings = true;
+  if (argc >= 2) {
+    std::string arg = ConsumeArg(argc, argv);
+    if (arg == "true") {
+      ignore_warnings = true;
+    } else if (arg == "false") {
+      ignore_warnings = false;
+    } else {
+      throw Usage{
+          "delete-app-profile: ignore-warnings parameter must be either"
```

Comment:
Maybe this should just print the same usage messages as above, with the complete set of arguments? Just a thought — if this error message is shown, it's unclear *which* argument is the `ignore-warnings` in this case.

Reply:
Yup.  Fixed.